### PR TITLE
Store notification platform

### PR DESCRIPTION
**Pull request by yafuquen**
## Description
- Store notification platform

## Testing
- [x] Added unit tests
- [x] Ran unit tests successfully
- [ ] Added documentation for public APIs and/or Wiki

### CODE REVIEW

File: TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/Models/PushFactor.swift [hunk 1]
```diff
@@ -34,4 +34,11 @@ struct PushFactor: Factor, Codable {
...
 struct Config: Codable {
   let credentialSid: String
+  var notificationPlatform: NotificationPlatform = .apn
 }
+
+public enum NotificationPlatform: String, Codable {
+  case apn
+  case none
+}
+
```
Comment (AuthyApps) [inline TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/Models/PushFactor.swift:44]: <table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/2fdfb7d085dc022bdb92dd7135b2ee5defe47bd1/TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/Models/PushFactor.swift#L44"></span>Files should have a single trailing newline.<br />
<code>trailing_newline</code> <code>PushFactor.swift:44</code></td>
    </tr>
  </tbody>
</table>

**Comment by AuthyApps** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/twilio/twilio-verify-ios/blob/2fdfb7d085dc022bdb92dd7135b2ee5defe47bd1/TwilioVerifySDKTests/TwilioVerify/Sources/Domain/Factor/FactorMapperTests.swift#L24">TwilioVerifySDKTests/TwilioVerify/Sources/Domain/Factor/FactorMapperTests.swift#L24</a> - Type body should span 200 lines or less excluding comments and whitespace: currently spans 201 lines<br />
<code>type_body_length</code> <code>FactorMapperTests.swift:24</code></td>
    </tr>
  </tbody>
</table>

## Code coverage
### Total coverage: **`95.16%`**
File | Coverage
-----|-----
TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/FactorMapper.swift | **`100.00%`**
TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/Models/PushFactor.swift | **`100.00%`**
> Powered by [Slather](https://github.com/SlatherOrg/slather)

<p align="right" data-meta="generated_by_UnitTests">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by AuthyApps** [comment]
## Code coverage
### Total coverage: **`95.20%`**
File | Coverage
-----|-----
TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/FactorMapper.swift | **`100.00%`**
TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/Models/PushFactor.swift | **`100.00%`**
> Powered by [Slather](https://github.com/SlatherOrg/slather)

<p align="right" data-meta="generated_by_UnitTests">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by mejiagarcia — APPROVED** [review]

**Review by AlejandroOrozco — APPROVED** [review]
LG 👍

**Pull request merged**